web/themes/custom/novel/templates/paragraphs/paragraph--simple-links.html.twig

```diff
@@ -1,7 +1,7 @@
 {% for item in content.field_link %}
   {% if item['#type'] == 'link' %}
-    <a href="{{ item['#url'].toString() }}" target="{{ item['#options'].attributes.target|default('_self') }}"
```

Comment:
*sigh* How did we end up here?

Unless you're doing something explicitly funky, `item` is a render array, and simply doing `{{ item }}` renders the link, a tag and everything (the link formatter takes care of the target).

Of course, you don't get the classes on the link, but `simple-link` doesn't seem to be defined (so why do it exist in the HTML?), and `arrow__hover--right-small` is presentation that's more related to the *context* of the link than the link itself (it should be handled by a `.paragraph--simple-links a` instead, if you ask me).

Reply:
I'm not sure what to do here. I tried deleting the file to see if it will render a link with `:` correctly. It doesn't. So I think we have to do it that way.


Alternatively, we should reconsider making a widget for the link, but that will require more time on the task and we have paragraphs that have a link as a wrapper where it would not be able to be used?

<img width="772" alt="image" src="https://github.com/user-attachments/assets/c1c59f47-180e-4f73-aae2-0614120af157">

Reply:
Deleting which file? I'm not following...

I *ought* to be enough just to use `{{ item }}` in the template in order to get the standard code to render the link, but that means we'll lose the classes. And while nobody will miss `simple-link`, the styling wont be right if the other disappears. And fixing the link styling is a bit out of scope for this PR.

But I do want to point out that we're spending time fixing things that's already fixed because we're not doing things as they're done in Drupal.

Reply:
I tried removing `web/themes/custom/novel/templates/paragraphs/paragraph--simple-links.html.twig` to see Drupal's default rendering, but that does not work for links with : (proxy links). 

Have I misrepresented something? 😅

Reply:
We should pair up on it in RL, so I can explain along the way rather than just issuing instructions.

Reply:
That will be nice!